google/cloud/spanner_v1/pool.py

```diff
@@ -229,12 +260,24 @@ def get(self, timeout=None):
         if timeout is None:
             timeout = self.default_timeout
 
+        start_time = time.time()
+        current_span = get_current_span()
+        span_event_attributes = {"kind": type(self).__name__}
+        current_span.add_event("Acquiring session", span_event_attributes)
         session = self._sessions.get(block=True, timeout=timeout)
+        span_event_attributes["session.id"] = session.session_id
 
         if not session.exists():
+            current_span.add_event(
+                "Session is not valid, recreating it", span_event_attributes
+            )
             session = self._database.session()
             session.create()
+            span_event_attributes["id"] = self._session_id
```

Comment:
What is the attribute we are using for session id? 
In some place above it was `session.id`, but here it is `id`. Can we follow the attribure naming consistently?